packages/documentation/src/stories/foundation/layout/Columns/columns.stories.ts

```diff
@@ -0,0 +1,255 @@
+      control: {
+        type: 'select',
+      },
+      options: [
+        'justify-content-start',
+        'justify-content-center',
+        'justify-content-end',
+        'justify-content-around',
+        'justify-content-between',
+        'justify-content-evenly',
+      ],
+      table: {
+        category: 'General',
+      },
+    },
+    offsetItem: {
+      name: 'Offset classes',
+      description: 'Increases the left margin of a column',
+      control: {
+        type: 'select',
+      },
+      options: [
+        'offset-md-1',
+        'offset-md-2',
+        'offset-md-3',
+        'offset-md-4',
+        'offset-md-5',
+        'offset-md-6',
+        'offset-md-7',
+        'offset-md-8',
+      ],
+      table: {
+        category: 'General',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+export const VerticalExample: Story = {
+  parameters: {
+    controls: { exclude: ['Horizontal Alignement', 'Offset classes'] },
+  },
+  render: (args: Args) => html`
+    <div class="container">
+      <div class="row-height row ${args.alignItems}">
+        <div class="col ${args.alignSelf === 'no self alignment' ? '' : args.alignSelf}">
+          Item 1
+        </div>
+        <div class="col">Item 2</div>
+        <div class="col">Item 3</div>
+      </div>
+    </div>
+  `,
+};
+
+export const HorizontalExample: Story = {
+  parameters: {
+    controls: { exclude: ['Align Items', 'Align Item 1', 'Offset classes'] },
+  },
+  render: (args: Args) => html`
+    <div class="container">
+      <div class="row ${args.justifyContent}">
+        <div class="col-4">Item 1</div>
+        <div class="col-4">Item 2</div>
+      </div>
+    </div>
+  `,
+};
+
+export const OrderExample: Story = {
+  render: () => html`
+    <div class="container">
+      <div class="row">
+        <div class="col">First in DOM, no order applied</div>
+        <div class="col order-5">Second in DOM, with a larger order</div>
+        <div class="col order-1">Third in DOM, with an order of 1</div>
+      </div>
+    </div>
+  `,
+};
+
+export const OrderMaxExample: Story = {
+  render: () => html`
+    <div class="container">
+      <div class="row">
+        <div class="col order-last">First in DOM, ordered last</div>
+        <div class="col">Second in DOM, unordered</div>
+        <div class="col order-first">Third in DOM, ordered first</div>
+      </div>
+    </div>
+  `,
+};
+
+export const OffsetExample: Story = {
+  parameters: {
+    controls: { exclude: ['Align Items', 'Align Item 1', 'Horizontal Alignement'] },
+  },
+  decorators: [
+    (story: StoryFn, { args, context }: StoryContext) => html`
+      ${story(args, context)}
+      <p class="mt-regular"><small>Resize the browser window to see changes.</small></p>
+    `,
+  ],
+  render: (args: Args) => html`
+    <div class="row">
+      <div class="col-md-4 ${args.offsetItem}">.col-md-4 .${args.offsetItem}</div>
```

Comment:
Add all available offset-classes to the controls (e.g. `.offset-1`, ... `.offset-11`).

```suggestion
      <div class="col-1 ${args.offsetItem}">.col-1 .${args.offsetItem}</div>
```